src/rtFileDownloader.h

```diff
@@ -83,6 +83,9 @@ class rtFileDownloadRequest
   bool cacheEnabled();
   void setDataIsCached(bool val);
   bool isDataCached();
+  void setReadCachedDataInChunks(bool val);
+  bool readCachedDataInChunksEnabled();
```

Comment:
Let's rename setReadCachedDataInChunks() and readCachedDataInChunksEnabled() APIs to setDeferCacheRead() and deferCacheRead().